From: Hector Palacios <hector.palacios@digi.com>
Date: Wed, 21 Dec 2016 16:28:05 +0100
Subject: [PATCH] gstimxv4l2: map /dev/video1 to /dev/fb0

On the i.MX6UL, the CSI camera is probed before the video PXP device.
In this case the camera is registered as /dev/video0 and the video device
is registered as /dev/video1.

In the mappings for video devices we must map /dev/video1 to /dev/fb0 so
that the imxv4l2sink can work with it.

Signed-off-by: Hector Palacios <hector.palacios@digi.com>

https://jira.digi.com/browse/DEL-3399
---
 libs/v4l2_core/gstimxv4l2.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libs/v4l2_core/gstimxv4l2.c b/libs/v4l2_core/gstimxv4l2.c
index 02cf68d9cddc..3ec3d6a41e18 100755
--- a/libs/v4l2_core/gstimxv4l2.c
+++ b/libs/v4l2_core/gstimxv4l2.c
@@ -208,6 +208,7 @@ static guint g_camera_format_PXP[] = {
 
 static IMXV4l2DeviceMap g_device_maps[] = {
   {"/dev/video0", FALSE, "/dev/fb0"},
+  {"/dev/video1", FALSE, "/dev/fb0"},
   {"/dev/video16", TRUE, "/dev/fb0"},
   {"/dev/video17", FALSE, "/dev/fb0"},
   {"/dev/video18", TRUE, "/dev/fb2"},
